diff --git a/finnel/src/schema.rs b/finnel/src/schema.rs
index fe3d4e7..606a78d 100644
--- a/finnel/src/schema.rs
+++ b/finnel/src/schema.rs
@@ -43,14 +43,14 @@ diesel::table! {
 
     records (id) {
         id -> BigInt,
         account_id -> BigInt,
         amount -> BigInt,
         currency -> Text,
-        operation_date -> Timestamp,
-        value_date -> Timestamp,
+        operation_date -> TimestamptzSqlite,
+        value_date -> TimestamptzSqlite,
         direction -> Text,
         mode -> Text,
         details -> Text,
         category_id -> Nullable<BigInt>,
         merchant_id -> Nullable<BigInt>,
     }
